.azure-pipelines/impacted_area_testing/get-impacted-area.yml

```diff
@@ -32,6 +32,12 @@ steps:
         FINAL_FEATURES=""
         break
 
+      # We can consider the scripts under `tests/scripts` as common scripts
+      # so the scope of PR testing is all test scripts
+      elif [[ "$FEATURE" == *tests/scripts* ]]; then
+        FINAL_FEATURES=""
+        break
+
```

Comment:
Can you provide like a white list here to make it more general? In case of other common scripts found in the future.

Reply:
Maybe we can provide a list for such common features.